crates/types/src/config/common.rs

```diff
@@ -225,12 +235,6 @@ pub struct CommonOptions {
     ///
     /// The retry policy for node network error
     pub network_error_retry_policy: RetryPolicy,
-
-    /// # Automatically provision number of configured partitions
-    ///
-    /// If this option is set to `false`, then one needs to manually write a partition table to
-    /// the metadata store. Without a partition table, the cluster will not start.
-    pub auto_provision_partitions: bool,
```

Comment:
How are we stopping the auto provisioning right now? It seems that we always configure a `cluster_configuration_seed` which will then create the `PartitionTable`?

Reply:
Partition table will be auto provisioned if cluster configuration is available in metadata store.

Reply:
If `bootstrap_num_partitions` is set to zero. Then we don't provision the partition table.